backend/src/apiserver/visualization/developer_guide.md

```diff
@@ -0,0 +1,118 @@
+# Python based visualizations guideline
```

Comment:
Would it make sense to name this file `README` instead of `developer_guide.md` so that GitHub displays its contents at the `/visualization` root by default?

Reply:
This is a great suggestion, I'll make that change!